### PR TITLE
[RU] Update and fix the `basics/basics` lesson

### PR DESCRIPTION
**Pull request by nscyclone**
This PR includes several fixes for the `basics/basics` lesson and the corresponding updates from #480.

### CODE REVIEW

File: ru/lessons/basics/basics.md [hunk 1]
```diff
@@ -13,7 +13,16 @@ lang: ru
 ## Настройка
 
 ### Установка Elixir
-Инструкции по установки для всех ОС есть на [официальном сайте](http://elixir-lang.org/install.html).
+
+Инструкции по установке для всех ОС есть на [официальном сайте](http://elixir-lang.org/install.html).
+
+После того, как Elixir установился, вы с лёгкостью можете проверить, какая именно версия была установлена.
```
Comment (brain-geek) [inline ru/lessons/basics/basics.md]: `установился` -> `установлен`

File: ru/lessons/basics/basics.md [hunk 2]
```diff
@@ -26,6 +35,17 @@ lang: ru
...
+Попробуем написать несколько простых выражений:
+
+    iex(1)> 2+3
+    5
+    iex(2)> 2+3 == 5
+    true
+    iex(3)> String.length("The quick brown fox jumps over the lazy dog")
+    43
+
+Не стоит волноваться, если пока вы поняли не все выражения. Надеемся, что вам понятен смысл.
```
Comment (brain-geek) [inline ru/lessons/basics/basics.md]: I think that `Надеемся, что вам понятен смысл.` can be safely deleted

File: ru/lessons/basics/basics.md [hunk 3]
```diff
@@ -97,6 +117,20 @@ iex> :true === true
 true
 ```
 
+Названия модулей в Elixir — тоже атомы. `MyApp.MyModule` — валидный атом, даже если такой модуль ещё не был объявлен..
```
Comment (brain-geek) [inline ru/lessons/basics/basics.md]: `тоже` -> `также` ?

Comment (brain-geek) [inline ru/lessons/basics/basics.md]: Ok, this one is also used in the next paragraph, so it's OK.

**Comment by brain-geek** [comment]
Hey, @nscyclone , I've looked through this and found some place for improvement.

File: ru/lessons/basics/basics.md [hunk 4]
```diff
@@ -118,6 +152,8 @@ iex> "foo\nbar"
 "foo\nbar"
 ```
 
+В Elixir также есть и более сложные типы данных. Мы узнаем о них больше, когда познакомимся с коллекциями и функциями.
```
Comment (brain-geek) [inline ru/lessons/basics/basics.md]: I think that after ommiting `также` this section will sound more natural

**Comment by brain-geek** [comment]
@nscyclone I've looked through it once again and found one last improvement. Otherwise it's awesome.

**Comment by nscyclone** [comment]
Thanks, @brain-geek, you reviewed this incredibly fast.

**Pull request merged**